silk/middleware.py

```diff
@@ -57,7 +56,19 @@ def process_request(self, request):
         return
 
 
-class SilkyMiddleware(MiddlewareMixin):
+class SilkyMiddleware(object):
```

Comment:
This shouldn't need to inherit from object anymore since we are now python 3.

Reply:
Thanks for the review, indeed, that is no longer needed.

Unfortunately I'm going on vacations and I can not fix that myself, I could gladly do that as soon as I'm back around August 15th